src/components/Header/Header.jsx

```diff
@@ -30,32 +31,59 @@ const navbarPages = [{
   path: '/become-the-gamer',
 }];
 
-const Header = ({ location }) => (
-  <div>
-    <Toolbar className={styles.header}>
-      <ToolbarGroup className={styles.verticalAlign}>
-        <AppLogo style={{ marginRight: 10 }} />
-      </ToolbarGroup>
-      <ToolbarGroup className={styles.verticalAlign}>
-        {navbarPages.map(page => (
-          <div key={page.name} className={styles.tabContainer}>
-            {page.external ?
-              <a href={page.path} className={styles.tab}>{page.name}</a> :
-                <Link to={page.path} className={styles.tab}>{page.name}</Link>}
-          </div>
-        ))}
-      </ToolbarGroup>
-      <ToolbarGroup className={styles.verticalAlign} style={{ marginLeft: 20 }}>
-        <ActionSearch style={{ marginRight: 8, opacity: '.6' }} />
-        <SearchForm location={location} />
-      </ToolbarGroup>
-      <ToolbarGroup className={styles.verticalAlign} style={{ marginLeft: 'auto' }}>
-        <AccountWidget />
-      </ToolbarGroup>
-    </Toolbar>
-  </div>
-);
+const Header = ({ location, width }) => {
+  const isTablet = width <= 820;
+  const isMobile = width <= 425;
+  const isTiny = width <= 320;
+  const toolbarPad = (isMobile && '8px') || (isTablet && '18px') || '22px';
+  const logoSize = (isTiny && '14px') || (isMobile && '16px');
 
-const mapDispatchToProps = () => ({});
+  const LogoGroup = () => (
+    <ToolbarGroup className={styles.verticalAlign}>
+      {isTablet && <BurgerMenu links={navbarPages} top={<AccountWidget />} />}
+      <AppLogo style={{ marginRight: 18 }} size={logoSize} />
+    </ToolbarGroup>
+  );
+
+  const LinkGroup = () => (
+    <ToolbarGroup className={styles.verticalAlign}>
+      {navbarPages.map(page => (
+        <div key={page.name} className={styles.tabContainer}>
+          {page.external ?
+            <a href={page.path} className={styles.tab}>{page.name}</a> :
+              <Link to={page.path} className={styles.tab}>{page.name}</Link>}
+        </div>
+      ))}
+    </ToolbarGroup>
+  );
+
+  const SearchGroup = () => (
+    <ToolbarGroup style={{ marginLeft: 20 }} className={styles.verticalAlign}>
+      <ActionSearch style={{ marginRight: 6, opacity: '.6' }} />
+      <SearchForm location={location} />
+    </ToolbarGroup>
+  );
 
-export default connect(null, mapDispatchToProps)(Header);
+  const AccountGroup = () => (
+    <ToolbarGroup className={styles.verticalAlign} style={{ marginLeft: 'auto' }}>
+      <AccountWidget />
+    </ToolbarGroup>
+  );
+
+  return (
+    <div>
+      <Toolbar style={{ padding: toolbarPad }} className={styles.header}>
+        <LogoGroup />
+        {!isTablet && (<LinkGroup />)}
+        <SearchGroup />
+        {!isTablet && (<AccountGroup />)}
+      </Toolbar>
+    </div>
+  );
+};
+
+const mapDispatchToProps = () => ({});
```

Comment:
I actually don't mind this as it's useful as a reminder that this is something that can be passed, and it's less code to change if we ever add mapDispatchToProps to this component.  Fine with either, though :)